src/components/Hero.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react'
+import key from '../assets/key.svg'
+import cmdK from '../assets/cmdK.svg'
+import starIcon from '../assets/starIcon.svg'
+import issueIcon from '../assets/issueIcon.svg'
+import { fetchWithSearch } from "../lib/supabase";
+import useDebounce from '../hooks/useDebounce'
+import humanizeNumber from "../lib/humanizeNumber";
+
+type PostResult = {
+  full_name: string;
+  user_id: number;
+  description: string;
+  stars: number;
+  issues: string;
+  contributions: { url: string; contributor: string; last_merged_at: string }[];
+};
+
+const Hero = () => {
+  const [searchTerm, setSearchTerm] = useState<string>("")
+  const debouncedSearchTerm: string = useDebounce<string>(searchTerm, 500);
+  const [results, setResults] = useState<PostResult[]>([]);
+  const [hasFocus, setFocus] = useState<boolean>(false);
+  
+  
+  useEffect(() => {
+    if (debouncedSearchTerm) {
+
+      fetchWithSearch("stars", 3, debouncedSearchTerm).then((results) => {
+        console.log(results)
+        setResults(results as unknown as []);
+      });
+    } else {
+      setResults([]);
+    }
+  }, [debouncedSearchTerm])
+  
+
+  return (
+    <div className='flex flex-col my-[95px] items-center'>
+        <div>
+          <h1 className='font-Lexend text-[48px] text-center text-lightSlate leading-[52.8px] tracking-[-6%] '>
```

Comment:
Hey, here's a suggestion of how I'd style this header

```suggestion
          <h1 className='font-Lexend text-5xl text-center text-lightSlate leading-tight tracking-tighter '>
```